docs/dietpi_docs_todo.md

```diff
@@ -223,3 +253,31 @@ Describe Backup/Restore:
 - via `dietpi-backup`
 - via different/other backup solutions
   E.g. see <https://www.linuxlinks.com/backup/>
+
+## 11. Optimized software
+
+### 11.1 Harmonization with <https://dietpi.com/dietpi-software.html>
+
+In <https://dietpi.com/dietpi-software.html> are icons of software titles as tiles displayed.  
+This should be consistent to our [optimized software list](https://dietpi.com/docs/dietpi_optimised_software/).
+
+Proposal: Vor every software title one tile. Add tiles to dietpi-software.html.
+
+#### Decision?
+
+### 11.2 Add section "how do I run..."
+
+In every chapter of "Optimized software" the section "How do I run DietPi-Software and install Optimised software ?" shall be added.
+
+#### Decision?
+
+## 12. Chapter "Advanced configuration"
+
+### 12.1 Add descriptions
+
+<https://dietpi.com/docs/dietpi_tools/#dietpi-software>
+
+- Someone to describe the tab "Search".
+- Someone to describe the tab "Log System".
+
+#### Decision?
```

Comment:
`Search` is currently explained within the `Software Optimised` tab currently. Since it also applies to software in the `Additional` menu, it indeed makes sense to make an own tab for it.